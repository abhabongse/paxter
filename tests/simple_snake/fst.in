john@"@"example.com
@<#"sarah@example.com"#>
@bold{Tom @italic{and} Jerry}!
@{italic("YAS")}
@bold#<{##<#{}#>##}>#
@tag[tag="u"]{works fine!}

My name is @name. I am @age years old!
@!{age = age + 1}
My name is @name. I am @age years old!
@bold[]{one}
