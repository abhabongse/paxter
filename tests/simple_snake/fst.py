name = "John"
age = 25
bold = lambda token: f"<b>{token}</b>"  # noqa: E731
italic = lambda token: f"<i>{token}</i>"  # noqa: E731
tag = lambda token, tag: f"<{tag}>{token}</{tag}>"  # noqa: E731
