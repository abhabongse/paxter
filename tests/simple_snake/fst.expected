john@example.com
sarah@example.com
<b>Tom <i>and</i> Jerry</b>!
<i>YAS</i>
<b>##<#{}#>##</b>
<u>works fine!</u>

My name is John. I am 25 years old!

My name is John. I am 26 years old!
